### DISCUSSION TITLE
Donations?

### DISCUSSION THREAD

**Discussion by dtcarls** [cwendt94/espn-api#459] [General]
You have been awesome at maintaining, expanding, and documenting this repo over the years @cwendt94 . I run GameDayBot.com for easy hosting of my open-source project https://github.com/dtcarls/fantasy_football_chat_bot which wouldn't be possible without your library. I try to give back by helping answering questions, but I'd also like to donate some for your hard work. I couldn't find a way to directly communicate with you on this platform, nor did I find a donation link in your bio or repository. Let us all know how we can return the favor, as I'm sure there are others out there wondering the same thing.
Thanks for what you do!

**Comment by dtcarls** [comment]
Just bumping this with the latest flurry of activity. Keep up the good work.

**Reply by cwendt94** [reply]
Thanks Dean, I do appreciate you helping out! I will look into adding a donation section!
